test/avocado/run-tests

```diff
@@ -103,6 +107,7 @@ def copy_avocado_logs(machine, title):
         if "exists" not in machine.execute(command=test_command,quiet=True):
             return
         machine.download_dir(avocado_results_dir, dir)
```

Comment:
I'm wondering if while we're at this, we could change `dir` to also be a proper subdir of `arg_attachments_dir` instead of the current working directory.